Sprint-2/errors/1.js

```diff
@@ -3,11 +3,14 @@
 // Why will an error occur when this program runs?
 // Try playing computer with the example to work out what is going on
 
+// Firstly, the decimalNumber variable has already been declared. Secondly, the function convertToPercentage has not been called correctly, instead console.log was used to log decimalNumber. By removing the const keyword and calling the function correctly allows the code to convert the given decimalNumber to a percentage.
+
 function convertToPercentage(decimalNumber) {
-  const decimalNumber = 0.5;
+  decimalNumber = 0.5;
   const percentage = `${decimalNumber * 100}%`;
 
   return percentage;
 }
 
-console.log(decimalNumber);
+let myPercentage = convertToPercentage()
```

Comment:
If the function expects a parameter, we should always pass a parameter in the function call. Otherwise the parameter will received the value `undefined`.

Reply:
Rectifies the problem ..and when I pass a parameter it doesn't return <undefined>